website/project/views/comment.py

```diff
@@ -21,33 +21,37 @@
 
 @file_updated.connect
 def update_file_guid_referent(self, node, event_type, payload, user=None):
-    if event_type == 'addon_file_moved' or event_type == 'addon_file_renamed':
-        source = payload['source']
-        destination = payload['destination']
-        source_node = Node.load(source['node']['_id'])
-        destination_node = node
-        file_guids = FileNode.resolve_class(source['provider'], FileNode.ANY).get_file_guids(
-            materialized_path=source['materialized'] if source['provider'] != 'osfstorage' else source['path'],
-            provider=source['provider'],
-            node=source_node)
-
-        if event_type == 'addon_file_renamed' and source['provider'] in settings.ADDONS_BASED_ON_IDS:
-            return
-        if event_type == 'addon_file_moved' and (source['provider'] == destination['provider'] and
-                                                 source['provider'] in settings.ADDONS_BASED_ON_IDS) and source_node == destination_node:
-            return
-
-        for guid in file_guids:
-            obj = Guid.load(guid)
-            if source_node != destination_node and Comment.find(Q('root_target._id', 'eq', guid)).count() != 0:
-                update_comment_node(guid, source_node, destination_node)
-
-            if source['provider'] != destination['provider'] or source['provider'] != 'osfstorage':
-                old_file = FileNode.load(obj.referent._id)
-                obj.referent = create_new_file(obj, source, destination, destination_node)
-                obj.save()
-                if old_file and not TrashedFileNode.load(old_file._id):
-                    old_file.delete()
+    if event_type not in ('addon_file_moved', 'addon_file_renamed'):
+        return  # Nothing to do
+
```

Comment:
Hm, so I thought the problem was that the listener wasn't getting registered because this file wasn't imported at startup. But I think the actual issue is that the waterbutler request to the `create_waterbutler_log` endpoint (which sends the `file_updated` signal) is failing.

I get this error in waterbutler:

```

aiohttp.errors.ClientOSError: [Errno None] Cannot connect to host localhost:5000 ssl:False [Can not connect to localhost:5000 [None]]
```

This might be a local dev environment issue--I did just update my waterbutler image and requirements--but I'm not sure. @cwisecarver or @chrisseto Are you able to reproduce this locally?

Steps:

- Create a project with a component
- Upload a file to the project
- Add a comment to the project
- Move the file to the component
- Go the the file's detail page

Expected: Comment is still on the file.
Actual: Comment is gone.